include/util/range_table.hpp

```diff
@@ -139,6 +139,18 @@ template <unsigned BLOCK_SIZE, bool USE_SHARED_MEMORY> class RangeTable
         sum_lengths = lengths_prefix_sum;
     }
 
+    void WriteARangeTable(osrm::storage::io::FileWriter &filewriter)
+    {
+        unsigned number_of_blocks = diff_blocks.size();
+
+        filewriter.WriteElementCount32(number_of_blocks);
```

Comment:
:+1: for iteratively removing all those four byte counts - there is no reason for saving four bytes but being prone to overflows and _some_ files having eight byte counts and _some_ having four byte counts.